### DISCUSSION TITLE
How to set Paperless-ngx configuration options

### DISCUSSION THREAD

**Discussion by DrSpaldo** [tteck/Proxmox#1941] [Help Needed] [answered]
Hi, I was wondering for the Paperless-ngx LXC how I would set configuration options, as per the docs; 
https://docs.paperless-ngx.com/configuration/

For example I want to enable to recursive options in the consumer folder:

PAPERLESS_CONSUMER_RECURSIVE=true
PAPERLESS_CONSUMER_SUBDIRS_AS_TAGS=true

**Comment by tteck** [comment, marked as answer]
`nano /opt/paperless/paperless.conf`

**Reply by DrSpaldo** [reply]
Great, thanks so much @tteck
